### PR TITLE
Multiplexing sample

### PR DESCRIPTION
**Pull request by EugeneIOs**
## Description of changes
-  Multiplexing sample

[IOSSDK-188]: https://verygoodsecurity.atlassian.net/browse/IOSSDK-188

### CODE REVIEW

File: demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+		cardNumber.becomeFirstResponder()
+
+		/// Use `VGSExpDateConfiguration` if you need to convert output date format
+		let expDateConfiguration = VGSExpDateConfiguration(collector: vgsCollect, fieldName: "data.attributes.details")
+		expDateConfiguration.type = .expDate
+		expDateConfiguration.inputDateFormat = .shortYear
+		expDateConfiguration.outputDateFormat = .longYear
+		expDateConfiguration.serializers = [VGSExpDateSeparateSerializer(monthFieldName: "data.attributes.details.month", yearFieldName: "data.attributes.details.year")]
+
+		/// Default .expDate format is "##/##"
```
Comment (dmytrokhl) [inline demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift:93]: remove the comment

File: demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	@IBAction func collectForMultiplexing(_ sender: Any) {
+		// hide kayboard
+		hideKeyboard()
+
+		// check if textfields are valid
+		vgsCollect.textFields.forEach { textField in
+			textField.borderColor = textField.state.isValid ? .lightGray : .red
+		}
+
+		// Multiplexing extra data
```
Comment (dmytrokhl) [inline demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift:151]: maybe add `Required` word

File: demoapp/demoapp/UseCases/MultiplexingSetup/Readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<p align="center">
+  <img src="./multiplexing-flow.png" alt="multiplexing-flow"/>
+</p>
+
+## How to run multiplexing sample
+
+1. Follow the instructions and install [Multiplexing](https://github.com/verygoodsecurity/multiplexing/blob/master/integration/README.md) app.
+
+2. Change `vaultId` to your vault id in `AppCollectorConfiguration` associated with Multiplexing setup.
```
Comment (dmytrokhl) [inline demoapp/demoapp/UseCases/MultiplexingSetup/Readme.md:16]: In Demo App, change...

**Review by dmytrokhl — APPROVED** [review]

File: demoapp/demoapp/UseCases/MultiplexingSetup/Readme.md (hunk 3 above)
Comment (EugeneIOs) [reply demoapp/demoapp/UseCases/MultiplexingSetup/Readme.md]: Done.

File: demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift (hunk 2 above)
Comment (EugeneIOs) [reply demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift]: Done.

File: demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift (hunk 1 above)
Comment (EugeneIOs) [reply demoapp/demoapp/UseCases/MultiplexingSetup/CollectMultiplexingViewController.swift]: Done.

**Pull request merged**